templates/zerver/help/about-streams-and-topics.md

```diff
@@ -33,7 +33,7 @@ stream.
 Users can also customize their stream settings; they can
 [pin](/help/pin-a-stream) or star important streams,
 [change the colors](/help/change-the-color-of-a-stream) of streams, and enable desktop
-notifications or muting for streams in order to create a better Zulip
+notifications or mute notifications for streams in order to create a better Zulip
```

Comment:
But I don't think this part is about muting notifications, it's about muting streams - this change seems thus unnecessary.